assets/src/components/map.tsx

```diff
@@ -18,6 +18,9 @@ import {
 } from "react-leaflet"
 // @ts-ignore
 import Control from "react-leaflet-control"
+// @ts-ignore
+import FullscreenControl from "react-leaflet-fullscreen"
+import "react-leaflet-fullscreen/dist/styles.css"
```

Comment:
Why is this (and the moduleNameMapper/cssStubber) necessary to import the css? Is there something different about this package compared to like the react-tooltip package that means we need to import the css explicitly? And if there is, is this import better here or in webpack/app.scss configuration?

Reply:
@skyqrose hmmmm, it didn't work without an explicit import before...and now it does! I guess I fixed it.